src/components/SecondaryPanes/index.js

```diff
@@ -30,6 +31,24 @@ const Accordion = React.createFactory(require("../shared/Accordion"));
 const CommandBar = React.createFactory(require("./CommandBar").default);
 require("./SecondaryPanes.css");
 
+type StartItems = {
```

Comment:
Can you explain the difference here between `StartItems` and `EndItems`? they look so similar except for `button` vs `buttons` and I'm not sure why we have a `button` attribute.